### ISSUE TITLE
0.8.x | Navigation Preset Player View Does Not Work

### ISSUE THREAD

**Issue by webmaster94** [earlSt1/vtt-navigation-presets#9]
I wanted to let you know that the setting to allow players to see the presets does not work in 0.8.x. All other features appear to be working in the new version except that.

**Comment by earlSt1** [comment]
Hi, I made some changes to fix issues with player presets previously. Can you try as a GM creating a new scene and not assigning it to a preset (so it stays in the Default preset)?

**Comment by webmaster94** [comment]
I have several scenes that are on the default preset.

**Comment by earlSt1** [comment]
Are the scenes visible to players?

**Comment by webmaster94** [comment]
Yes, all scenes that have navigation, including the ones that are in presets are visible to players so long as they have permission to the scene. I believe this is the default Foundry behavior. Of course, this makes a bit of an unwieldly list and takes up unnecessary space. This was kind of why I was using this module.

**Comment by earlSt1** [comment]
Righto, perhaps its a different issue then. Are you getting any errors in the F12 console when you login as a player?

**Comment by webmaster94** [comment]
This appears to be the only error relating to this module:
![image](https://user-images.githubusercontent.com/85364225/120899426-4cbc4b80-c5f5-11eb-8cbb-8f60dbe199d6.png)

**Comment by earlSt1** [comment]
Ok can you also send me the output of this macro (logged in as player):
```js
let activePreset = game.settings.get('navigation-presets','active-preset')
console.log(activePreset)
```

**Comment by webmaster94** [comment]
Result: undefined

**Comment by earlSt1** [comment]
Weird, i'm getting this issue too... I'll push an update to the module once i figure out why it's doing this.

**Comment by webmaster94** [comment]
Understood. Thanks for looking into it.

**Comment by earlSt1** [comment]
Gah, proper facepalm moment there. Forgot to change some code relating to the last bugfix.

Can you try again by uninstalling and using this manifest: https://raw.githubusercontent.com/earlSt1/vtt-navigation-presets/master/test/module.json

**Comment by webmaster94** [comment]
That appears to have fixed it.

**Comment by earlSt1** [comment]
Great, fix included in v1.0.10, update and you should get it
